easybuild/easyblocks/m/mxnet.py

```diff
@@ -97,21 +101,26 @@ def extract_step(self):
 
         for srcdir in [d for d in os.listdir(self.builddir) if d != os.path.basename(self.mxnet_src_dir)]:
             submodule, _, _ = srcdir.rpartition('-')
-            newdir = os.path.join(self.mxnet_src_dir, submodule)
+
+            # the package mkldnn was renamed to oneDNN since this easyblock was written
```

Comment:
I'm still confused by this. This easyblock makes no assumptions about the name of this package.
If MXNet's build scripts assuming the old name for this submodule? If so

```suggestion
            # MXNet assumes the old for oneDNN which was mkldnn
```